graylog2-web-interface/src/views/components/searchbar/time-range-filter/TimeRangeFilter.tsx

```diff
@@ -62,12 +63,13 @@ const TimeRangeFilter = ({
   noOverride = false,
   value = NO_TIMERANGE_OVERRIDE,
   onChange,
-  validTypes,
+  validTypes = undefined,
   position = 'bottom-start',
-  className,
+  className = undefined,
   showPresetDropdown = true,
   limitDuration,
   withinPortal = true,
+  submitOnPresetChange = true,
```

Comment:
Just a general thought, if we use `undefined` here as well we only have to maintain the default value once where we actually make use of the value, in `TimeRangeFilterButtons`.